documentation/asset-management/topics/asset-properties.rst

```diff
@@ -0,0 +1,93 @@
+.. include:: /asset-management/media.rst
+.. _Important-asset-properties-doc:
+
+****************
+Asset properties
+****************
+Assets can have many properties and many property types. These properties can be updated using one of the following methods:
+
+1. Discovery
+2. Inherited from catalog data
+3. Inherited from the parent asset
+4. Filled by a third-party integration
+5. Inputted by the user
+
+Asset properties can have an impact on how the system behaves. Some critical properties, especially identifying properties like Serial Number and GUID, cannot be changed once they have been populated by the auto-discovery process.
+
+.. note:: Certain properties such as "Rack Design" are copied from a property set on the parent location. This is done on asset creation only.
+
+Asset properties are usually grouped under the following types:
+
+====
+Base
+====
+These are the properties of Name, Location, Type, Manufacturer and Model. The base properties answer the **what** and **where** questions of an asset.
+
+Base properties of an asset are usually detected by the auto-discovery process. However, a user can always update an asset manually or override the data filled in by auto-discovery.
+
+For manually added assets, these properties can be set individually or in bulk using the **Import** feature.
+
+=======
+Contact
+=======
+These are location-specific properties. They contain, the address, latitude, longitude and the location's human contact information.
+
+The Address property is always checked using the Google Maps API and once a match is found, the validated address is stored along with the latitude and longitude reported by the API.
+
+This data is used to plot the location on the map layout and is also used to read the available weather forecast.
+
+=======
+Cooling
+=======
+These properties apply to locations and racks. It controls the behavior of the Delta-T calculation for racks that have linked temperature sensors. New racks will clone their parent location's setting. Users can override this value per rack or in bulk.
```

Comment:
This may be improved by replacing "It controls...", with "The Rack Delta-T Calculation Orientation property controls..."

Reply:
Addressed, thanks!